homematicip/aio/device.py

```diff
@@ -425,3 +425,6 @@ class AsyncAccelerationSensor(AccelerationSensor, AsyncDevice):
         return await self._connection.api_call(
             *super().set_notification_sound_type(soundType, isHighToLow, channelIndex)
         )
+
+class AsyncGarageDoorModuleTormatic(GarageDoorModuleTormatic, AsyncDevice):
+    """ HMIP-MOD-TM (Garage Door Module Tormatic) """
```

Comment:
async version of send_door_command is missing

Reply:
async command added